frontend/src/_includes/partials/audio-actions.html

```diff
@@ -0,0 +1,72 @@
+<div class="mb-1 flex space-x-1">
+  <!-- Download -->
+  <a
+    href="{{ fileUrl }}"
+    class="inline-flex items-center space-x-1"
+    title="Download file for listening offline"
+  >
+    <svg
+      class="h-full w-4 stroke-current"
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+    >
+      <path
+        stroke-linecap="round"
+        stroke-linejoin="round"
+        stroke-width="2"
+        d="M4 16v1a3 3 0 003 3h10a3 3 0 003-3v-1m-4-4l-4 4m0 0l-4-4m4 4V4"
+      />
+    </svg>
+    <span class="hidden md:inline">Download</span>
+  </a>
+  <!-- Feedback -->
+  <a
+    href="{{ feedbackUrl }}"
+    target="_blank"
+    class="inline-flex items-center space-x-1"
+    title="Help us improve! Give feedback about the sound quality, title, contents, language, etc of this file"
+  >
+    <svg
+      class="h-full w-4 stroke-current"
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+    >
+      <path
+        stroke-linecap="round"
+        stroke-linejoin="round"
+        stroke-width="2"
+        d="M8 10h.01M12 10h.01M16 10h.01M9 16H5a2 2 0 01-2-2V6a2 2 0 012-2h14a2 2 0 012 2v8a2 2 0 01-2 2h-5l-5 5v-5z"
+      />
+    </svg>
+    <span class="hidden md:inline">Feedback</span>
+  </a>
+  <!-- Share -->
+  <a
+    href="/audios/{{ audioObj.fileId }}/"
+    class="inline-flex cursor-pointer items-center space-x-1"
+    x-data="webshare('{{ audioObj.title | replace: "'", "\\'" | escape }}', '/audios/{{ audioObj.fileId }}/')"
```

Comment:
Make both `fileId` and `title` separate parameters of this partial. Don't pass `audioObj` as a whole.

Reply:
done